source/extensions/geoip_providers/maxmind/geoip_provider.cc

```diff
@@ -27,7 +27,8 @@ GeoipProviderConfig::GeoipProviderConfig(
                                                  : absl::nullopt),
       anon_db_path_(!config.anon_db_path().empty() ? absl::make_optional(config.anon_db_path())
                                                    : absl::nullopt),
-      scope_(scope), stat_name_set_(scope.symbolTable().makeSet("Maxmind")),
+      stats_scope_(scope.createScope("")),
```

Comment:
Should the stat_prefix be used here, instead of an empty string?

Reply:
Fixed